src/http.ts

```diff
@@ -422,7 +443,7 @@ export async function initReport(
 
   const res = await makeRequest<InitUploadResponseDto>(config);
   if (res.success) return { type: 'success', value: res.body };
-  return generateError<GetAnalysisErrorCodes>(res.errorCode, GET_ANALYSIS_ERROR_MESSAGES, 'initReport');
+  return generateError<GetAnalysisErrorCodes>(res.errorCode, REPORT_ERROR_MESSAGES, 'initReport');
```

Comment:
```suggestion
  return generateError<ReportErrorCodes>(res.errorCode, REPORT_ERROR_MESSAGES, 'initReport');
```

Reply:
Out of curiosity. Don't we have extra error messages when we initiate a report? Like we have it below on line 462

Reply:
For now, we don't any that is worth propagating to the client side, but if this changes we can always add it here 😁